meeting-notes/2019/Q1/2019-01-02--gui-and-in-web-browsers-weekly.md

```diff
@@ -0,0 +1,115 @@
+# GUI & In Web Browsers Weekly Sync 2019-01-02
+
+-   **Moderator & Notetaker:** _&lt;none, it was a casual post-holiday call with surprisingly good attendance :)>_
+-   **Attendees:**
+
+    -   _@terichadbourne_
+    -   _@lidel_
+    -   _@jimpick_
+    -   _@olizilla_
+    -   _@hacdias_
+
+  
+
+
+**Moderator checklist**
+
+-   Ensure that there is a notetaker
+-   Start recording
+-   Call for additional agenda items
+
+  
+
+
+**Agenda**
+
+_General discussions, decisions, etc._
+
+-   Start with Team Updates and Demos
+
+  
+
+
+**Team Updates**
+
+@lidel
+
+-   Done:
+    -   Holidays :-)
+    -   Discussions around accessing API of remote IPFS node - [in-web-browsers/issues/137](https://github.com/ipfs/in-web-browsers/issues/137) 
+        -   @Gozala shared some relevant ideas in  
+            [Progressive peer-to-peer web applications (PPWA)](https://via.hypothes.is/https://gozala.hashbase.io/posts/Native%20talk.html)  and tested initial assumptions in [in-web-browsers/issues/137#comment-450530548](https://github.com/ipfs/in-web-browsers/issues/137#issuecomment-450530548) 
+    -   Switched Companion to modern Clipboard API, PDF and SVG work again - [ipfs-companion/pull/649](https://github.com/ipfs-shipyard/ipfs-companion/pull/649)
+    -   Added a warning about mixed-content when gateway URL is not a Secure Context - [ipfs-companion/pull/650](https://github.com/ipfs-shipyard/ipfs-companion/pull/650) 
+    -   Started discussion about the future of DNSLink cache - [ipfs-companion/pull/646](https://github.com/ipfs-shipyard/ipfs-companion/pull/646#issuecomment-450879014)  + [go-ipfs/issues/5884](https://github.com/ipfs/go-ipfs/issues/5884) 
+-   Next:
+    -   Finalize OKRs
```

Comment:
Let me know if I can help :)

Reply:
@daviddias draft is in [the 'wb' spreadsheet](https://docs.google.com/spreadsheets/d/1BtOfd7s9oYO5iKsIorCpsm4QuQoIsoZzSz7GItE-9ys/#gid=755202447), if anything looks off would appreciate  feedback as inline comments there or https://github.com/ipfs/team-mgmt/pull/804 :)

Reply:
I just went through the KRs and posted a couple of comments. I believe there is room to enhance them with more real/current users interaction